src/components/deltabargraph.js

```diff
@@ -107,19 +131,18 @@ function DeltaBarGraph({timeseries, arrayKey}) {
   );
 }
 
-export default React.memo(DeltaBarGraph, () => {
-  return true;
-});
+export default React.memo(DeltaBarGraph);
+// export default React.memo(DeltaBarGraph, () => {
+//   return true;
+// });
```

Comment:
I wouldn't comment this all out. It could lead to unnecessary rendering. Better to compare the props to see if they're unchanged.